biomage/experiment/utils.py

```diff
@@ -116,3 +118,35 @@ def download_S3_json(s3_obj, key, filepath):
     s3_obj.download_file(local_file)
 
     set_modified_date(file_location=local_file, date=s3_obj.last_modified)
+
+
+def get_cognito_username(email):
+    client = boto3.client("cognito-idp")
+    user_name = client.admin_get_user(Username=email, UserPoolId="eu-west-1_sVQL15Yxu")[
+        "Username"
+    ]
+
+    return user_name
+
+
+def add_user_to_rbac(user_name, cfg):
+    if "rbac_can_write" in cfg:
+        cfg["rbac_can_write"]["SS"].append(user_name)
```

Comment:
what would happen if we try to add a username that's already been added? Will it add it twice, or will the fact that the values in `rbac_can_write` are a set prevent from this to happen?